action.yml

```diff
@@ -36,47 +36,50 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: imranismail/setup-kustomize@v2
-      with:
-        kustomize-version: ${{ inputs.kustomize-version }}
+  - uses: imranismail/setup-kustomize@v2
+    with:
+      kustomize-version: ${{ inputs.kustomize-version }}
 
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        token: ${{ inputs.token }}
-        ref: ${{ inputs.branch }}
-        repository: ${{ inputs.repo }}
-        path: infra
+  - name: Checkout
+    uses: actions/checkout@v4
+    with:
+      token: ${{ inputs.token }}
+      ref: ${{ inputs.branch }}
+      repository: ${{ inputs.repo }}
+      path: infra
 
-    - name: Kustomize
-      shell: bash
-      run: |
-        IFS=$'\n'
-        lines="${{ inputs.images }}"
-        for line in $lines; do
-          IFS=',' read -r path image_name <<< "$line"
-          cd "${{ github.workspace }}/infra/$path"
+  - name: Kustomize
+    shell: bash
+    run: |
+      wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/bin/yq
+      chmod +x /usr/bin/yq
```

Comment:
This shouldn't be necessary as [the ubuntu-latest runner already includes yq](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#tools).

```suggestion
```